kong/db/schema/entities/clustering_data_planes.lua

```diff
@@ -30,5 +30,14 @@ return {
                       default = "unknown",
                     }
     },
+    { plugin_versions = { type = "array",
+                          elements = { type = "record",
```

Comment:
As this is an array of records, perhaps just call it `plugins`. Perhaps later on it starts to contain more than just `version` information. And similar adjustments to migrations.

Reply:
In that case should we make this a record? So plugins.versions is then array of {name, version}.

Reply:
I think what Aapo is saying is leaving everything as it is, but just naming it `plugins`.

The elements of your current `plugin_versions` contain two fields: `name` and `version`. What he's saying is that if in the future we want to extend that with other things that are plugin versions (for example to include `kong_version_min` or `kong_version_max`) it would work.

I don't think we should make this `plugins.versions` an array and then have other arrays named `plugins.kong_version_mins`.

(If we were programming in Pascal though, that would be our only option :) )